atc/db/check_factory.go

```diff
@@ -139,8 +139,20 @@ func (c *checkFactory) Resources() ([]Resource, error) {
 	var resources []Resource
 
 	rows, err := resourcesQuery.
-		Join("(select DISTINCT(resource_id) FROM job_inputs) ji ON ji.resource_id = r.id").
-		Where(sq.Eq{"p.paused": false}).
+		LeftJoin("(select DISTINCT(resource_id) FROM job_inputs) ji ON ji.resource_id = r.id").
+		LeftJoin("(select DISTINCT(resource_id) FROM job_outputs) jo ON jo.resource_id = r.id").
+		Where(sq.Or{
+			sq.And{
+				sq.Eq{"p.paused": false},
```

Comment:
Might be clearer to extract this into a chained `Where` since it's the same in both branches.

Reply:
Can't extract this into a chained where because the squirrel library only `AND`'s where's together, there's no way to `OR` two `where()` calls. That's why it's all stuffed into one `where()` and duplicated :(

Reply:
This is also why I refactored the `resourcesQuery` into `activeResourcesQuery`, so I could start with a query with no `where()`.